### PR TITLE
sql: add more descriptions about create table parameters

### PR DESCRIPTION
**Pull request by hfxsd**
### What is changed, added or deleted? (Required)

As tittle

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [ ] v7.2 (TiDB 7.2 versions)
- [x] v7.1 (TiDB 7.1 versions)
- [ ] v7.0 (TiDB 7.0 versions)
- [ ] v6.6 (TiDB 6.6 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: auto-increment.md [hunk 1]
```diff
@@ -12,6 +12,8 @@ aliases: ['/docs-cn/dev/auto-increment/']
 >
 > 使用 `AUTO_INCREMENT` 可能会给生产环境带热点问题，因此推荐使用 [`AUTO_RANDOM`](/auto-random.md) 代替。详情请参考 [TiDB 热点问题处理](/troubleshoot-hot-spot-issues.md#tidb-热点问题处理)。
 
+在 [CREATE TABLE](/sql-statements/sql-statement-create-table.md) 语句中的 `AUTO_INCREMENT` 参数也可以用于指定自增字段初始值。
```
Comment (hfxsd) [inline auto-increment.md:15]:
```suggestion
也可以使用 [CREATE TABLE](/sql-statements/sql-statement-create-table.md) 语句中的 `AUTO_INCREMENT` 参数指定自增字段初始值。
```

File: auto-random.md [hunk 2]
```diff
@@ -12,6 +12,8 @@ aliases: ['/docs-cn/dev/auto-random/','/docs-cn/dev/reference/sql/attributes/aut
 
 关于如何在高并发写入场景下调优 TiDB，请参阅 [TiDB 高并发写入场景最佳实践](/best-practices/high-concurrency-best-practices.md)。
 
+在 [CREATE TABLE](/sql-statements/sql-statement-create-table.md) 语句中的 `AUTO_RANDOM_BASE` 参数，也可以用来指定 `AUTO_RANDOM` 自增部分的初始值，该参数可以被认为属于内部接口的一部分，对于用户而言请忽略。
```
[the remark this answers is not in the document]
Comment (hfxsd) [reply auto-random.md:15]: 照搬的这里的原文 https://docs.pingcap.com/zh/tidb/stable/sql-statement-create-table#create-table  可以保留现状

**Review by ran-huang — APPROVED** [review]

**Pull request merged**